src/services/auth.service.ts

```diff
@@ -14,29 +14,42 @@ class AuthService extends Repository<UserEntity> {
     super();
   }
 
-  private createToken(data: BaseUserResponseDTO): TokenData {
-    const time = 3600 * 100;
-    const expiresIn = String(time);
-    return { expiresIn, token: sign({ ...data }, SECRET_KEY, { expiresIn }) };
+  public createToken(data: BaseUserResponseDTO, type: 'access' | 'refresh'): TokenData {
+    const [expiresIn, secretKey] = (() => {
+      if (type === 'access') {
+        return ['600000', ACCESS_TOKEN_SECRET_KEY];
+      }
+      return ['30 days', REFRESH_TOKEN_SECRET_KEY];
+    })();
+    return { expiresIn, token: sign({ ...data }, secretKey, { expiresIn }) };
   }
 
-  private createCookie(tokenData: TokenData): string {
-    return `Authorization=${tokenData.token}; HttpOnly; Max-Age=${tokenData.expiresIn};`;
+  public createCookie(tokenData: TokenData, type: 'access' | 'refresh'): string {
```

Comment:
nit: you can destructure the properties of tokenData

Reply:
done